.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+### Issues
```

Comment:
@misscs If the screenshot is highlighting a specific problem, that might be best in the original linked issue.